block_context.go

```diff
@@ -7,18 +7,29 @@ package slack
 type ContextBlock struct {
 	Type     MessageBlockType `json:"type"`
 	BlockID  string           `json:"block_id,omitempty"`
-	Elements []BlockObject    `json:"elements"`
+	Elements ContextElements  `json:"elements"`
 }
 
-// blockType returns the type of the block
-func (s ContextBlock) blockType() MessageBlockType {
+// BlockType returns the type of the block
+func (s ContextBlock) BlockType() MessageBlockType {
 	return s.Type
 }
 
-// NewContextBlock returns a newinstance of a context block
-func NewContextBlock(blockID string, elements ...BlockObject) *ContextBlock {
+type ContextElements struct {
+	ContextElementSet []MixedElement
+}
+
+// NewContextElements is a convenience method for generating ContextElements
+func NewContextElements(contextElements []MixedElement) ContextElements {
```

Comment:
s/[]MixedElement/...MixedElement/

Reply:
actually does this method need to exist? or can we just use `...MixedElement` for NewContextBlock

Reply:
will look into this tomorrow ^ :eyes:

Reply:
indeed does not need to exist